backend/game/shop/shop.py

```diff
@@ -0,0 +1,63 @@
+import json
+import random
+
+from typing import List, Dict, TextIO, Tuple
+from math import exp, floor
+
+class Shop:
+
+    path_to_units_file: str = './../data/units.json'
+
+    def __init__(self):
+        with open(self.path_to_units_file, 'r') as units_f:
+            self.units_list = self.read_json(units_f)
+        self.units_amount = len(self.units_list)
+        self.add_prices()
+
+    def read_json(self, file: TextIO) -> List[Dict]:
+        return json.load(file)
+
+    def add_prices(self):
+        for unit in self.units_list:
+            price = self.calculate_price(unit)
+
+    def calculate_price(self, unit: Dict) -> int:
+        stats_sum = 0
+        for stat in unit['statistics']:
+            stats_sum += unit['statistics'][stat]
+        stats_sum -= unit['statistics']['base_hp']
```

Comment:
Bubasz będzie smutny że hp się nie liczy :cry:

Reply:
Uznałem że HP jest czynnikiem za mocno zmieniającym wartość sumaryczną, więc nie liczę.
Ewentualnie do przemyślenia by jakoś to HP dzielić (np /10) i wtedy dodawać.

Reply:
Jestem za dzieleniem :+1: